scripts/teardown.sh

```diff
@@ -15,14 +15,19 @@
 # limitations under the License.
 
 set -e
+set -x
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 # shellcheck source=scripts/istio.env
 source "$ROOT/scripts/istio.env"
 
-kubectl delete deploy,svc --all -n default
-kubectl delete deploy,svc --all -n istio-system
-kubectl delete deploy,svc --all -n vm
+gcloud beta container clusters update "$ISTIO_CLUSTER" \
+    --update-addons=Istio=DISABLED \
+    --project "$ISTIO_PROJECT" \
+    --zone "$ZONE"
+
+kubectl delete ns vm --ignore-not-found=true
+kubectl delete ns bookinfo --ignore-not-found=true
```

Comment:
We need to wait till this are cleaned up as well

Reply:
I am gonna merge but we need another PR in for us to wait on the services to delete